### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by maasencioh**
* create documentation
* add missing tests

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+extends: 'eslint-config-cheminfo'
+parserOptions:
+  sourceType: module
+env:
+  jest: true
```
Comment (targos) [inline .eslintrc.yml]: please remove the env property. Are you using an outdated version of the cheminfo generator? It was fixed a month ago

**Review by targos — DISMISSED** [review]

**Comment by targos** [comment]
this changes the supported node versions -> semver-major

**Comment by stropitek** [comment]
LGTM except that I don't understand your change in the value similarity. It looks like you are completely changing the behavior of it. The order matters in your implementation but not in the previous one.

**Comment by targos** [comment]
I didn't realize this was changing behavior. Can you back out those changes and open another PR once this refactoring has been merged?

File: .eslintrc.yml (hunk 1 above)
Comment (maasencioh) [reply .eslintrc.yml]: cheminfo generator is not working alone with me, so I was using directly the yeoman generator, but it is indeed an outdated one

**Review by targos — APPROVED** [review]

**Comment by maasencioh** [comment]
@stropitek the order is not really important, what I checked was that the response was not aligned with the expected answer of the test case, that's why I did change it

**Review by stropitek — APPROVED** [review]

**Comment by stropitek** [comment]
The idea was to calculate a similarity based on the presence and absence of values/keywords in 2 lists. If it is present in list 1 but not in list 2, `M10 += 1`, if in list 2 but not list 1, `M01 += 1` and if present in both `M11 += 1`.

The test case therefore seems correct to me (watch out for the `1` and `l` that are different). However this method needs a better documentation for sure! And if you find a better name for it I'm in.
```js
var values1 = ['1', 1, 'x', true, 'true', []];
var values2 = [true, false, 'y', 'l', 5, '1', []];
const M01 = 5, M10 = 4, M11 = 2;
```

**Comment by maasencioh** [comment]
@stropitek now I understand, but as far as I understand the order matters, for example, for me this somehow should be correct:

```js
var values1 = [1, 1, 0];
var values2 = [1, 0, 1];
const M01 = 1, M10 = 1, M11 = 1;
const sim !== 1;
```

**Comment by stropitek** [comment]
Tanimoto is kind of a synonym for Jaccard, and is basically the intersection over the union of sets. The original Tanimoto paper used bitmaps, but if each bit represents the presence / absence of a  distinct feature or a keyword, it is equivalent to Jaccard.

**Comment by targos** [comment]
@maasencioh the function to use with arrays of 0 and 1 is the bit array similarity.

**Comment by maasencioh** [comment]
Ok, indeed, it was inconsistent to me the fact that `[true, false, false],[true, true, false]` will give different results using values and binaries, but as far as I read you are right, thank you for review that part!

**Pull request merged**